clients/src/test/java/org/apache/kafka/clients/admin/KafkaAdminClientTest.java

```diff
@@ -263,7 +267,7 @@ public void testGenerateClientId() {
         Set<String> ids = new HashSet<>();
         for (int i = 0; i < 10; i++) {
             String id = KafkaAdminClient.generateClientId(newConfMap(AdminClientConfig.CLIENT_ID_CONFIG, ""));
-            assertTrue("Got duplicate id " + id, !ids.contains(id));
+            assertFalse("Got duplicate id " + id, ids.contains(id));
```

Comment:
All changes in this file are side cleanups except in `testIncrementalAlterConfigs` and `testAlterConfigsUponNotController`.